src/composables/useClient.ts

```diff
@@ -10,6 +10,22 @@ export function useClient() {
   const auth = getInstance();
   const { notify } = useNotifications();
 
+  function successNotification(keyword: string) {
+    let notification;
+
+    if (keyword === 'delete-proposal')
```

Comment:
Why not a switch statement?

Reply:
```js

const notification = (() => {
  switch(keyword): 
    case 'delete-proposal': 
       return ['green', t('notify.proposalDeleted')]
    // ...
})()

```

Reply:
Never us it, do you think it's a good idea here?

Reply:
I have been using a pattern these days to avoid `let`.

`const variable = (() => {return "hi"}())`

instead of 
```
let variable
function changeVariable() {
  variable = "hi"
}
```

In this example, I combined both this pattern and switch statement.

Reply:
@bonustrack yes agree, did it like that now